### ISSUE TITLE
Typographic error

### ISSUE THREAD

**Issue by rmdobservations** [DjangoGirls/tutorial-extensions#65]
In views.py, I get an error in the redirect phrase for post_publish.

I think that "pk=pk" should read "pk=post.pk" .
Unless I am missing something.

def post_publish(request, pk):
    post = get_object_or_404(Post, pk=pk)
    post.publish()
    return redirect('post_detail', pk=pk)

greetings,
Rose

**Comment by helenst** [comment]
Hi @rmdobservations 
Do you mean on the last line? I think in that case either should work (`post.pk` and `pk` will have the same value). What is the error you are seeing?

**Comment by rmdobservations** [comment]
If they both work that is ok.I have since found another error so maybe that was the real cause.  I noticed that in the other post_*  defs, there was a "post.pk". I cannot check the post.pk vs. pk until I fix what I broke yesterday. I do not yet understand entirely what is going on inside Django. The usage of urls has been a challenge. But I am slowly going through your tutorials and am making progress. 
Thanks!
Rose
